Templates/gRPC-Template/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Don't think this is needed :D

Reply:
We need a source of truth for formatter. Otherwise, your editor config can override this one.